src/azul/openapi/params.py

```diff
@@ -0,0 +1,80 @@
+from typing import (
+    Union
+)
+
+from azul.openapi import schema
+from azul.openapi.schema import (
+    TYPE
+)
+from azul.types import (
+    JSON,
+    PrimitiveJSON,
+)
+
+
+# Path params cannot be optional
+def path(name: str, type_: TYPE, **kwargs: PrimitiveJSON) -> JSON:
+    """
+    Returns a JSON `parameters` entry for an endpoint parameter specified via
```

Comment:
Slightly imprecise and a bit awkward.

```suggestion
    Returns an OpenAPI `parameters` specification of a URL path parameter.
```

See if you can improve the other docstrings accordingly. I haven't look at them in detail.